_posts/2016-06-29-standard-getting-started.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "The Need for bin/start"
+date: 2016-06-29 9:31:00
+categories: ["maintenance", "conventions"]
+author: "etagwerker"
+---
+
+Getting started with a new project should be as simple as possible, even for
+someone who is not super technical. As a maintainer, you **must** make sure that
+anyone can clone your project and start it in a few minutes.
```

Comment:
@etagwerker Maybe you could use `get it up and running`, `get started` or something to that effect, instead of `start it`? It sounds a bit ambiguous.